src/Compilers/VisualBasic/Portable/Symbols/FieldSymbol.vb

```diff
@@ -417,12 +419,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
-        Private ReadOnly Property IFieldSymbol_IsRequired As Boolean Implements IFieldSymbol.IsRequired
```

Comment:
>Private ReadOnly Property IFieldSymbol_IsRequired As Boolean Implements IFieldSymbol.IsRequired [](http://example.com/codeflow?start=8&length=95)

Please follow the pattern and keep this member. Adjust implementation instead.

Reply:
`IsReadOnly` in this type follows the same pattern I used.

Reply:
It is quite obvious that ```IsReadonly``` is an outlier and should be fixed too.